### PR TITLE
Router

### PR DESCRIPTION
**Pull request by timon94**
create modules and router

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,104 +1,37 @@
...
-
-            <!--slider-->
-            <div id="carousel" class="carousel slide">
-                <!--indicatorSlides-->
-                <ol class="carousel-indicators">
-                    <li class="active" data-target="#carousel" data-slide-to="0"></li>
-                    <li data-target="#carousel" data-slide-to="1"></li>
-                    <li data-target="#carousel" data-slide-to="2"></li>
-                </ol>
+    <body ng-app="sportApp" class="head-body">
```
Comment (NE05KY) [inline index.html]: Why do you need 'head-body' class?

File: js/app.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
-(function(){
-    angular.module('sportApp', ['viewBroadcastsModule']);
+(function() {
+    'use strict';
+
+    angular.module('sportApp', ['broadcasts.app', 'profile.app', 'ngRoute']);
```
Comment (NE05KY) [inline js/app.js]: You dont need '.app' postfix in your module names, simplify it.

File: index.html (hunk 1 above)
Comment (timon94) [inline index.html]: done

File: js/app.js (hunk 2 above)
Comment (timon94) [inline js/app.js]: done

**Pull request merged**